[CN-1372]: add note to TLS document about how to update TLS (#251)
hazelcast/hazelcast-platform-operator-docs: docs/modules/ROOT/pages/tls.adoc

----
include::ROOT:example$/management-center-tls.yaml[]
----

> NOTE: To update TLS certificates, it is required to delete and recreate the Hazelcast/Management Center custom resource.

Reviewer: ```suggestion
NOTE: To update TLS certificates, you need to delete and recreate the Hazelcast/Management Center custom resource.
```
Is it either the Hazelcast or MC resource you need to delete or both? The / is ambiguous here...

Author: TLS certificates are applicable for both CRs. So whichever you created you should delete it and recreate.

Reviewer: What do you think of update suggestion then?

Author: I think this should be enough: `NOTE: To update TLS certificates, you need to delete and recreate the appropriate custom resource`, WDYT?

Reviewer: Yep, ship it!
